tools/debug_tool/debug_server.py

```diff
@@ -0,0 +1,56 @@
+from flask import Flask, render_template, request, send_file, abort
+from flask_cors import CORS
+from requests import get
+import os
+
+app = Flask(__name__)
+app.config["CORS_ORIGINS"] = ["*"]
+CORS(app, support_credentials=True)
+
+KOTEKAN_ADDRESS = "http://localhost:12048"
```

Comment:
This should be an optional argument to the python script, with `http://localhost:12048` as the default.  It will likely be the case that the user will want to run the server locally, but debug a kotekan instance on a remote server.